test/java/src/com/ibm/streamsx/topology/test/python/SubscribeSPLDictTest.java

```diff
@@ -95,6 +98,20 @@ public void testSubscribeSPLDictMap() throws Exception {
             	}
         	}
 
+        	{
+        		Set<?> ex =spl.getSet("si32");
+            	JSONArray pya = (JSONArray) json.get("si32");
+            	assertEquals(ex.size(), pya.size());
+            	
```

Comment:
Can the contents of the set be tested by converting the list to a set and then assert equals on the sets?